A32NX/html_ui/Pages/A32NX_Core/A32NX_GroundReference.js

```diff
@@ -0,0 +1,12 @@
+class A32NX_GroundReference {
+    constructor() {
+        console.log('A32NX_GroundReference constructed');
+    }
+    init() {
+        console.log('A32NX_GroundReference init');
+    }
+    update(_dTime, _core) {
+        const groundReference = Simplane.getAltitude() - Simplane.getAltitudeAboveGround();
+        SimVar.SetSimVarValue("L:A32NX_ALTIMETER_GROUND_REFERENCE", "feet", groundReference);
+    }
+}
```

Comment:
This file does not have a trailing newline, similar to what you did to the CHANGELOG above. Maybe check your editor settings? (although I'm not a contributor, so I'm not aware of a style guide).

Reply:
Same as the other modules